packages/react-native-aztec/src/AztecView.js

```diff
@@ -122,8 +122,6 @@ class AztecView extends Component {
 	}
 
 	_onFocus( event ) {
-		AztecInputState.notifyInputChange();
-
```

Comment:
So for Aztec we pass the following prop:

```js
onFocus={ this._onAztecFocus }
```

For Android, it doesn't do anything and for iOS it calls `_onPress` which calls `focus`:

https://github.com/WordPress/gutenberg/blob/695a65bccba66afa9878d7ee14a360163323a932/packages/react-native-aztec/src/AztecView.js#L207-L215

I think we are covering all cases, what do you think?

Reply:
Good point, I've checked how the focus/blur events are handled in the `AztecView` component, and confirm that we're covering all the cases ✅ .

Specifically, the focus event is always triggered by either pressing the `AztecView` via [the `onPress` handler](https://github.com/WordPress/gutenberg/blob/695a65bccba66afa9878d7ee14a360163323a932/packages/react-native-aztec/src/AztecView.js#L200-L205) or programmatically via the `focus` function ([example](https://github.com/WordPress/gutenberg/blob/695a65bccba66afa9878d7ee14a360163323a932/packages/rich-text/src/component/index.native.js#L839)). So, we don't need to call `notifyInputChange` here as it will be done in `AztecInputState`.